easybuild/easyconfigs/o/OpenMolcas/OpenMolcas-22.10-intel-2022a.eb

```diff
@@ -0,0 +1,48 @@
+easyblock = 'CMakeMake'
+
+name = 'OpenMolcas'
+version = '22.10'
+
+homepage = "https://gitlab.com/Molcas/OpenMolcas"
+description = "OpenMolcas is a quantum chemistry software package."
+
+toolchain = {'name': 'intel', 'version': '2022a'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://gitlab.com/Molcas/OpenMolcas/-/archive/v%(version)s/']
+sources = ["OpenMolcas-v%(version)s.tar.gz"]
+patches = ['OpenMolcas-21.06_mcpdft_deps.patch']
+checksums = [
+    {'OpenMolcas-v22.10.tar.gz': 'd1d3d6c9bd69486c36f5285a51ac18e75c03540c5b33751855e94abc764d9658'},
+    {'OpenMolcas-21.06_mcpdft_deps.patch': '708763e060aa0174bbe9c0cc336b509fbf492066465908dd919452836a7a793a'},
+]
+
+builddependencies = [('CMake', '3.23.1')]
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('HDF5', '1.12.2'),
+    ('GlobalArrays', '5.8.1'),
+]
+
+build_shared_libs = True
+
+preconfigopts = 'GAROOT=$EBROOTGLOBALARRAYS'
+configopts = '-DLINALG=MKL -DMKLROOT=$MKLROOT '
+configopts += '-DOPENMP=ON -DTOOLS=ON -DFDE=ON'
+configopts += '-DMPI=ON -DGA=ON '
```

Comment:
And the same issue here with the lack of a space at the end of the second line.

Reply:
Fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/16770/commits/41bf0895e03a05391cbd1510136f6a9cb213ff36